azure-cli.pyproj

```diff
@@ -922,6 +922,17 @@
     <Content Include="command_modules\azure-cli-webapp\requirements.txt" />
   </ItemGroup>
   <ItemGroup>
+    <Interpreter Include="..\..\rc6\">
```

Comment:
Nope. I needed it while Derek's RC6 branch was still being worked on.

---
In reply to: [79086251](https://github.com/Azure/azure-cli/pull/903#discussion_r79086251) [](ancestors = 79086251)